eo-parser/src/test/java/org/eolang/parser/EoSyntaxTest.java

```diff
@@ -47,6 +47,7 @@
  *
  * @since 0.1
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@Yanich96 I believe, you don't need that line. Could you please, remove the line and run the following command:
```
mvn qulice:check -Pqulice
```
I did it myself and the linter said nothing about methods number.

Reply:
@volodya-lombrozo Test failed : PMD: eo-parser/src/test/java/org/eolang/parser/EoSyntaxTest.java[50-250]: This class has too many methods, consider refactoring it. (TooManyMethods)
 I used this command.

Reply:
@volodya-lombrozo This line will be deleted in the next pr, when I will transfer hash code in file attributes.